www_src/pages/page/page.jsx

```diff
@@ -164,6 +166,10 @@ var Page = React.createClass({
           state.currentElementId = id;
         }
         this.setState(state);
+
+        setTimeout(function() {
```

Comment:
I had an issue with the menu taking a few ms to hide, during which you could again click the buttons